src/test/fuzz/connman.cpp

```diff
@@ -25,7 +25,7 @@ FUZZ_TARGET_INIT(connman, initialize_connman)
 {
     FuzzedDataProvider fuzzed_data_provider{buffer.data(), buffer.size()};
     SetMockTime(ConsumeTime(fuzzed_data_provider));
-    CAddrMan addrman;
+    CAddrMan addrman(/* deterministic */ false, /* consistency_checks */ 0);
```

Comment:
In the previous incarnation of this PR which did not pass `consistency_checks` explicitly: `CAddrMan addrman(/* deterministic */ false)` it would have equaled to `DEFAULT_ADDRMAN_CONSISTENCY_CHECKS`.

In the latest incarnation (24515fee60) `0` is passed explicitly.

This contains the assumption that `DEFAULT_ADDRMAN_CONSISTENCY_CHECKS` is `0` which is true now, but could change in the future.

I think it is good to keep using `DEFAULT_ADDRMAN_CONSISTENCY_CHECKS` as default value instead of `0`. That is, either call `CAddrMan addrman(/* deterministic */ false, DEFAULT_ADDRMAN_CONSISTENCY_CHECKS)` in all places or restore the default second argument, which defaults to `DEFAULT_ADDRMAN_CONSISTENCY_CHECKS`.

Reply:
It's the other way around. The previous version implicitly assumed that the `DEFAULT_ADDRMAN_CONSISTENCY_CHECKS` was 0. This line is explicitly saying that we want an addrman without consistency checks.

If we used `DEFAULT_ADDRMAN_CONSISTENCY_CHECKS` explicitly in the call sites and that value changes in future (unlikely I think, since we don't want these debug checks enabled in production), then all call sites would have to be checked to make sure that the change in behavior is what is wanted. For example, we certainly wouldn't want to enable the consistency checks in benchmarks, since it'd totally skew the measurements.

Reply:
I see, makes sense, thanks for the elaboration!